bindata/deployment/frr/metallb-frr.yaml

```diff
@@ -374,6 +379,57 @@ spec:
             readOnlyRootFilesystem: true
           command:
             - /speaker
+        {{ if .IsOpenShift }}
```

Comment:
Can we have this driven by an "metrics under rbac" knob as opposed to the "IsOpenshift" one?
Exposing metrics via rbac is a common problem so other deployments could benefit from it.

Reply:
done